images/tests/Dockerfile.rhel

```diff
@@ -6,6 +6,7 @@ RUN make build WHAT=cmd/openshift-tests; \
     cp /go/src/github.com/openshift/origin/_output/local/bin/linux/$(go env GOARCH)/openshift-tests /tmp/build/openshift-tests
 
 FROM registry.svc.ci.openshift.org/ocp/4.0:cli
+RUN chmod g+w /etc/passwd
```

Comment:
Add this into the other RUN line after the other commands

Reply:
> Add this into the other RUN line after the other commands

Done with a296c55c0e -> ca35cd633a.